server/src/main/java/org/apache/cassandra/sidecar/server/Server.java

```diff
@@ -302,11 +303,15 @@ protected void validate()
      */
     protected Future<String> scheduleInternalPeriodicTasks(String deploymentId)
     {
-        periodicTaskExecutor.schedule(new HealthCheckPeriodicTask(vertx,
-                                                                  sidecarConfiguration,
-                                                                  instancesMetadata,
-                                                                  executorPools,
-                                                                  metrics));
+        HealthCheckPeriodicTask healthCheckPeriodicTask = new HealthCheckPeriodicTask(
+        sidecarConfiguration,
+                                                                   instancesMetadata,
+                                                                   executorPools,
+                                                                   metrics);
```

Comment:
Can we fix the formatting here?
```suggestion
        HealthCheckPeriodicTask healthCheckPeriodicTask = new HealthCheckPeriodicTask(sidecarConfiguration,
                                                                                      instancesMetadata,
                                                                                      executorPools,
                                                                                      metrics);
```

Reply:
Done